test/CheckoutSdkTest/Customers/CustomersClientTest.cs

```diff
@@ -21,7 +21,7 @@ public CustomersClientTest()
                 .Returns(_authorization);
 
             _configuration = new Mock<CheckoutConfiguration>(_sdkCredentials.Object,
-                Environment.Sandbox, _httpClientFactory.Object);
+                Environment.Sandbox, null, _httpClientFactory.Object);
```

Comment:
Does this have to be `null`?  I'm not sure the type here, but could it be some other kind of empty type?
Or is the purpose of the test to ensure that it succeeds when that value is `null`?

Reply:
Replacing with another constructor in the configuration. Thanks